core/DataTable/Filter/ColumnCallbackAddMetadata.php

```diff
@@ -45,10 +45,6 @@ public function __construct($table, $columnsToRead, $metadataToAdd, $functionToA
     {
         parent::__construct($table);
 
-        if (!is_array($columnsToRead)) {
-            $columnsToRead = array($columnsToRead);
-        }
-
         $this->columnsToRead      = $columnsToRead;
```

Comment:
I don't remember the exact time but it was certainly an improvement worth it otherwise I wouldn't have done it. It is actually no longer used after I replaced it afterwards with `MetadataCallbackAddMetadata`. I reckon we can still leave it there for improved performance in other use cases.